### PR TITLE
--name is not the right option expected

### PR DESCRIPTION
**Pull request by malachma**
changed --same to --disk 
as this is the Parameter expected

### CODE REVIEW

File: fstab.sh [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+mount -o bind /dev dev/
+mount -o bind /dev/pts dev/pts/
+
+chroot /mnt/rescue-root
+
+mv /etc/fstab{,.org}
+cat /etc/fstab.org | awk '/\/ /{print}' >> /etc/fstab
+cat /etc/fstab.org | awk '/\/boot /{print}' >> /etc/fstab
+cat /etc/fstab
+
```
Comment (sribs) [inline fstab.sh]: I would suggest to rename the file as `chroot-ops.sh` and create another file `fstab.sh` which performs fstab-specific operations and finally sourcing them like this
`chroot-ops.sh`
```bash
case "$1" in 
    fstab)
    . ./fstab.sh
    ;;
    install-wallinuxagent)
    . ./install-waagent.sh
    ;;
    *)
    echo "Usage: [fstab|install-walinuxagent]"
    ;;
esac
```

**Comment by malachma** [comment]
Hi Sriharsha, the fstab.sh is a single file. Which gets invoked via the use of the custom script Extension.
I think this is at the Moment the best way to handle different scenearious as we Can call them all seperate or also as Collection if needed. See my changes in the rescue.sh to see what I did so far

**Comment by sribs** [comment]
I have approved the changes for rescue.sh

**Comment by sribs** [comment]
You can merge the PR

**Pull request merged**